### PR TITLE
Move changes to the fork

### PR DESCRIPTION
**Pull request by khrapovs**
https://github.com/chintai-platform/ai-anomaly-Simulator-AMLSim is essentially a fork of https://github.com/IBM/AMLSim without actually creating a fork, but rather cloning and pushing to our own private organisation repo.

In this PR I have moved all the changes from https://github.com/chintai-platform/ai-anomaly-Simulator-AMLSim to https://github.com/chintai-platform/AMLSim.

### CODE REVIEW

File: scripts/convert_logs.py [hunk 1]
```diff
@@ -549,6 +575,7 @@ def __init__(self, conf, sim_name=None, fake=None):
         self.log_file = os.path.join(self.work_dir, output_conf["transaction_log"])
         self.in_acct_file = input_conf["accounts"]  # Account list file from the transaction graph generator
         self.group_file = input_conf["alert_members"]  # Alert account list file from the transaction graph generator
+        self.group_file_1 = input_conf["normal_models"]
```
[the remark this answers is not in the document]
Comment (khrapovs) [reply scripts/convert_logs.py:578]: Please, rename

File: scripts/convert_logs.py [hunk 2]
```diff
@@ -680,35 +708,35 @@ def convert_acct_tx(self):
                     elif output_item['name'] == 'last_name':
                         output_row[output_index] = us_gen.last_name_male() if gender == "Male" else us_gen.last_name_female()
 
-                    elif output_item['name'] == 'street_addr':
-                        output_row[output_index] = street_address
+                    # elif output_item['name'] == 'street_addr':
```
Comment (khrapovs) [inline scripts/convert_logs.py:711]: revert to the original

File: scripts/convert_logs.py [hunk 3]
```diff
@@ -863,6 +891,41 @@ def convert_alert_members(self):
             output_row = self.schema.get_alert_acct_row(alert_id, reason, account_id, account_id, is_sar,
                                                         model_id, schedule_id, bank_id, **attr)
             writer.writerow(output_row)
+    
+
+    def convert_non_alert_members(self):
```
Comment (khrapovs) [inline scripts/convert_logs.py:896]: most likely can be removed

File: scripts/transaction_graph_generator.py [hunk 4]
```diff
@@ -389,8 +389,7 @@ def load_account_list_raw(self):
         end_range = get_positive_or_none(self.default_end_range)
         default_model = self.default_model if self.default_model is not None else 1
 
-        self.attr_names.extend(["first_name", "last_name", "street_addr", "city", "state", "zip",
-                                "gender", "phone_number", "birth_date", "ssn", "lon", "lat"])
+        self.attr_names.extend(["first_name", "last_name", "city", "gender",  "birth_date"])
```
Comment (khrapovs) [inline scripts/transaction_graph_generator.py:392]: revert to the original

File: scripts/transaction_graph_generator.py [hunk 5]
```diff
@@ -399,16 +398,16 @@ def load_account_list_raw(self):
             idx_aid = name2idx["uuid"]
             idx_first_name = name2idx["first_name"]
             idx_last_name = name2idx["last_name"]
-            idx_street_addr = name2idx["street_addr"]
+            # idx_street_addr = name2idx["street_addr"]
```
Comment (khrapovs) [inline scripts/transaction_graph_generator.py:401]: revert to the original

File: scripts/transaction_graph_generator.py [hunk 6]
```diff
@@ -418,18 +417,19 @@ def load_account_list_raw(self):
                 if row[0].startswith("#"):  # Comment line
                     continue
                 aid = row[idx_aid]
-                first_name = row[idx_first_name]
-                last_name = row[idx_last_name]
-                street_addr = row[idx_street_addr]
+                # first_name = row[idx_first_name]
```
Comment (khrapovs) [inline scripts/transaction_graph_generator.py:420]: revert to the original

File: scripts/transaction_graph_generator.py [hunk 7]
```diff
@@ -442,9 +442,7 @@ def load_account_list_raw(self):
                 else:
                     end = -1
 
-                attr = {"first_name": first_name, "last_name": last_name, "street_addr": street_addr,
-                        "city": city, "state": state, "zip": zip_code, "gender": gender,
-                        "phone_number": phone_number, "birth_date": birth_date, "ssn": ssn, "lon": lon, "lat": lat}
+                attr = {"first_name": first_name, "last_name": last_name, "city": city, "gender": gender, "birth_date": birth_date}
```
Comment (khrapovs) [inline scripts/transaction_graph_generator.py:445]: revert

File: scripts/transaction_graph_generator.py [hunk 8]
```diff
@@ -1277,12 +1275,12 @@ def write_normal_models(self):
         output_file = os.path.join(self.output_dir, self.out_normal_models_file)
         with open(output_file, "w") as wf:
             writer = csv.writer(wf)
-            column_headers = ["modelID", "type", "accountID", "isMain", "isSAR", "scheduleID"]
+            column_headers = ["alertID", "modelID", "type", "accountID", "isMain", "isSAR", "scheduleID"]
```
Comment (khrapovs) [inline scripts/transaction_graph_generator.py:1278]: New column is redundant. Use `isSAR` instead.

File: scripts/transaction_graph_generator.py [hunk 9]
```diff
@@ -1277,12 +1275,12 @@ def write_normal_models(self):
...
             writer.writerow(column_headers)
             
             for normal_model in self.normal_models:
                 for account_id in normal_model.node_ids:
-                    values = [normal_model.id, normal_model.type, account_id, normal_model.is_main(account_id), False, 2]
+                    values = [-1, normal_model.id, normal_model.type, account_id, normal_model.is_main(account_id), False, 2]
```
Comment (khrapovs) [inline scripts/transaction_graph_generator.py:1283]: revert

**Review by khrapovs — COMMENTED** [review]
@akankshatoshniwal

**Comment by PhillipHamnett** [comment]
@khrapovs Is this still in progress?

**Pull request closed without merging**